client/src/components/publications.jsx

```diff
@@ -1,6 +1,29 @@
 import React, { Component } from 'react';
+import { navigate } from '@reach/router';
+import { ApiService } from '../services';
 
 class MyPublications extends Component {
+  editPublications() {
+    navigate('/pub-display');
+  }
+
+  deletePubs(value) {
+    let publication = value;
+    ApiService.delete('pubs/myPubs', {
+      name: publication.name,
+      twitterHandle: publication.twitterHandle,
+    })
+      .then((response) => {
+        if (response.statusText === 'OK') {
+          this.props.getUser();
+          navigate('/userProfile');
+        }
+      })
+      .catch((error) => {
```

Comment:
This is where you would handle 409 errors. Any error code 4xx or 5xx will fall into the `catch`.

Reply:
I still don't seem to get it - let's go through on Zoom next time we talk... Thanks